src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js

```diff
@@ -214,6 +219,24 @@
             });
         }
 
+        function loadPinDimensions() {
+            vm.pinDimensions = vm.dimensions.filter(function (item) {
+                return item.pin === true;
+            });
+            var headerSettings = {
+                showFSFilter: filterParametersService.getFiltersCount() == 0 ? false : true,
+                showPinFilter: vm.pinDimensions.length == 0 ? false : true
+            }
+            if (vm.pinDimensions.length > 0) {
+                vm.showPinFilter = true;
+                $rootScope.$broadcast("flairbiApp:toggle-headers-filters", headerSettings);
```

Comment:
$rootScope.$broadcast("flairbiApp:toggle-headers-filters", headerSettings); is duplicate. its being call in both the case